### PR TITLE
feat: Follow up for success drop notification

### LINKED ISSUE #6252: Follow up for success drop notif
Let's solve few things and add a Twitter redirect to the success notification
image

### Visual changes
- text align center
- make the container bigger, like in the Figma - updated there

### Functional 
- let's make the counter dynamic so that it will change every second. 29,28,....
- adding a new sentence with Twitter redirect text link.
- it will redirect you to Twitter with a tweet preset. Maybe @vikiival can do some magic and display an item/image that was already minted
- copy: 
  				
					Just minted an exclusive NFT with unlockable items on @Kodadot! 🎉 So excited to add this unique collectible to my collection. Don't miss your chance!

- I guess I need some help here from @JustLuuuu 🥺

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Needs Design check

- @exezbcz please review

## Context

- [x] Closes #6252
- [ ] Requires deployment <snek/rubick/worker>

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)

## Screenshot 📸

- [x] My fix has changed UI

tested on `/stmn/unlockable?redesign=true`

image

image

twitter
image

### <samp>🤖 Generated by Copilot at f4f1c7b</samp>

Added feature flag and mock code for unlockable NFTs and improved UX for minting and sharing them. The changes affect the `UnlockableContainer` and `UnlockableLoader` components in `components/collection/unlockable`.

### <samp>🤖 Generated by Copilot at f4f1c7b</samp>

> _Sing, O Muse, of the skillful code that changed the face of NFTs_
> _How the clever developers added a feature flag and mock code_
> _To test the new design of the `UnlockableContainer` component_
> _And make it easier to mint and share the hidden treasures of art_

### CODE REVIEW

**Comment by Jarsen136** [comment]
Interesting, the build always failed.

**Comment by Jarsen136** [comment]
I guess the failed error is related to the build cache because it builds successfully on my self-deploy env.

Preview env:
https://issue-6252--thriving-bunny-03a5e8.netlify.app/stmn/unlockable?redesign=true

**Comment by preschian** [comment]
> I guess the failed error is related to some wired build cache because it builds successfully on my self-deploy env.

on cypress/e2e should be fixed with this https://github.com/kodadot/nft-gallery/pull/6265
on netlify, I think error on their side at that time. seems like fixed for now

**Review by preschian — APPROVED** [review]

File: components/collection/unlockable/UnlockableLoader.vue [hunk 1]
```diff
@@ -3,11 +3,20 @@
...
       <NeoIcon class="close-icon" icon="close" @click.native="closeLoading" />
       <img :src="unloackableLoaderImg" />
-      <div class="is-flex is-flex-direction-column is-align-items-center px-7">
-        <div class="has-text-weight-bold my-2">Congratulations</div>
-        <div>
-          Get ready for the big reveal! Your NFT will be visible in just
-          <span class="has-text-weight-bold">30 seconds</span> on your profile.
+      <div
+        class="is-flex is-flex-direction-column is-align-items-center px-5 has-text-centered is-capitalized">
+        <div class="has-text-weight-bold mb-2">Congratulations</div>
```
Comment (roiLeo) [inline components/collection/unlockable/UnlockableLoader.vue:8]:
```suggestion
        <div class="has-text-weight-bold mb-2">{{ $t('mint.success') }}</div>
```

File: components/collection/unlockable/UnlockableLoader.vue [hunk 2]
```diff
@@ -3,11 +3,20 @@
...
+        <div class="">
```
Comment (roiLeo) [inline components/collection/unlockable/UnlockableLoader.vue:9]:
```suggestion
        <div>
```

**Review by roiLeo — APPROVED** [review]
- i18n 
- redesign

✅ otherwise lgtm

**Comment by exezbcz** [comment]
Hey, hey, I have a few changes.

First, I would change the copy; the first one I gave you must be corrected. 
Before: "Get ready for the big reveal! Your NFT will be visible in 30 seconds." 
After: "Get ready for the big reveal! Your NFT is visible now."

bold: 30 seconds, visible now

- >let's make the counter dynamic so that it will change every second. 29,28,....

- It will be good if you can share your success on Twitter in the waiting time - that means the share link will be visible even if you are waiting for the nft to be ready. @vikiival said it is possible to display the image you have selected in the tweet. The link in the tweet will redirect to the unlockable page. Please let me know if I am wrong.

Thanks!

**Comment by Jarsen136** [comment]
> * It will be good if you can share your success on Twitter in the waiting time - that means the share link will be visible even if you are waiting for the nft to be ready. @vikiival said it is possible to display the image you have selected in the tweet. The link in the tweet will redirect to the unlockable page. Please let me know if I am wrong.
> 
> Thanks!

I have explained the reason why we could not attach the image auto on this comment https://github.com/kodadot/nft-gallery/issues/6252#issuecomment-1595002796
You could also check this answer https://stackoverflow.com/questions/18321649/twitter-web-intent-to-post-image. We could not display the image auto by the redirect link, except the user attaches it manually. 


Except for the changes in the image display, the rest has been completed ✅ 
image
image

**Comment by exezbcz** [comment]
@Jarsen136 ah oki, nevermind, my bad :D 

Could you please make the countdown dynamic so that every second it lowers and lowers? It will give the user a more interactive feeling.


I belive that is the last thing

**Comment by Jarsen136** [comment]
> Could you please make the countdown dynamic so that every second it lowers and lowers? It will give the user a more interactive feeling.
> 

The minting time is estimated, which means it's not equal to 30 seconds or a specific number. 


> * I think that the cross has disappeared

It's related to the icon display on the self-deploy preview env. After merging this PR, I bet the cross icon will be back as usual.
I have tested it on my local env.
image

**Comment by exezbcz** [comment]
> The minting time is estimated, which means it's not equal to 30 seconds or a specific number.
> 
> >

Yes :D I wanted to make it an actual countdown, not a still number. It's better for the user. I know it stays the same for the minting time etc. 

> It's related to the icon display on the self-deploy preview env. After merging this PR, I bet the cross icon will be back as usual.
> I have tested it on my local env.

oki nice!

**Comment by Jarsen136** [comment]
> > The minting time is estimated, which means it's not equal to 30 seconds or a specific number.
> > >
> 
> Yes :D I wanted to make it an actual countdown, not a still number. It's better for the user. I know it stays the same for the minting time etc.
> 

Sorry, but I do not think it's a good idea to make it a countdown. For example, if we set the estimated to 30s, the minting may be finished in a time longer than 30s, in this case, the number would change from 30 to 0 seconds and stay at zero, then the user may feel more confused. WDYT?

**Comment by exezbcz** [comment]
@Jarsen136 yes, agree, but we set the 30s, so there is a really high probability that it's going to be finalized.

**Comment by Jarsen136** [comment]
> @Jarsen136 yes, agree, but we set the 30s, so there is a really high probability that it's going to be finalized.

Alright, let me make a countdown for it.

**Comment by exezbcz** [comment]
@Jarsen136 Yes, if it wont work well in the future :D my fault 


Thanks, sir!

**Comment by Jarsen136** [comment]
> @Jarsen136 Yes, if it wont work well in the future :D my fault
> 
> Thanks, sir!

You're welcome. The countdown should work now. Please check it.

File: components/collection/unlockable/UnlockableContainer.vue [hunk 3]
```diff
@@ -201,6 +203,14 @@ onMounted(async () => {
   const res = await getLatestWaifuImages()
   imageList.value = res.result.map((item) => item.output)
   resultList.value = res.result
+
+  // for test
+  if (redesign.value) {
```
Comment (vikiival) [inline components/collection/unlockable/UnlockableContainer.vue:208]: So remove?

**Review by vikiival — CHANGES_REQUESTED** [review]
translations otherwise oki

File: components/collection/unlockable/UnlockableContainer.vue (hunk 3 above)
Comment (Jarsen136) [reply components/collection/unlockable/UnlockableContainer.vue]: removed

File: components/collection/unlockable/UnlockableLoader.vue [hunk 4]
```diff
@@ -70,7 +72,7 @@ const displaySeconds = computed(() => {
 
 const twitterText = computed(
   () =>
-    'Just minted an exclusive NFT with unlockable items on @Kodadot! 🎉 So excited to add this unique collectible to my collection. Don\'t miss your chance! \n\n https://kodadot.xyz/stmn/unlockable'
+    "Just minted an exclusive NFT with unlockable items on @Kodadot! 🎉 So excited to add this unique collectible to my collection. Don't miss your chance! \n\n https://kodadot.xyz/stmn/unlockable"
```
Comment (vikiival) [inline components/collection/unlockable/UnlockableLoader.vue:75]: use single pls

Comment (Jarsen136) [reply components/collection/unlockable/UnlockableLoader.vue]: done

**Comment by prury** [comment]
I'm not testing this one since the redesign variable already got removed and @exezbcz has thoroughly tested

**Review by vikiival — APPROVED** [review]

**Pull request merged**